services/QuillLMS/app/queries/adapters/csv/admin_premium_data_export.rb

```diff
@@ -6,43 +6,66 @@ module Adapters
   module Csv
     class AdminPremiumDataExport
       class UnhandledColumnError < StandardError; end
+      class BigQueryResultMissingRequestedColumnError < StandardError; end
 
-      ORDERED_COLUMNS = %i(
-        student_name
-        student_email
-        completed_at
-        activity_name
-        activity_pack
-        activity_session_id
-        score
-        timespent
-        standard
-        tool
-        school_name
-        classroom_grade
-        teacher_name
-        classroom_name
-        student_id
-        classroom_id
-      )
-
-      def self.to_csv_string(bigquery_result, columns = ORDERED_COLUMNS)
+      ORDERED_COLUMNS = {
+        student_name:     'Student Name',
+        student_email:    'Student Email',
+        completed_at:     'Completed Date',
+        activity_name:    'Activity',
+        activity_pack:    'Activity Pack',
+        score:            'Score',
+        timespent:        'Time Spent (Mins)',
+        standard:         'Standard',
+        tool:             'Tool',
+        school_name:      'School',
+        classroom_grade:  'Grade',
+        teacher_name:     'Teacher',
+        classroom_name:   'Class',
+      }
+
+      def self.to_csv_string(bigquery_result, columns = ORDERED_COLUMNS.keys)
         sym_columns = columns.map(&:to_sym)
-        validate_input!(bigquery_result)
+        validate_input!(bigquery_result, sym_columns)
 
         CSV.generate do |csv|
-          csv << sym_columns.map(&:to_s)
+          csv << human_displayable_csv_headers(sym_columns)
           bigquery_result.each do |row|
-            csv << sym_columns.map { |key| row[key] }
+            csv << sym_columns.map { |key| format_cell(key, row[key]) }
           end
         end
       end
 
-      def self.validate_input!(bigquery_result)
+      def self.human_displayable_csv_headers(sym_columns)
+        sym_columns.map{|col| ORDERED_COLUMNS[col] }
+      end
+
+      def self.format_cell(sym_column, value)
+        case sym_column
+        when :completed_at
+          value&.strftime("%F")
+        when :timespent
+          (value / 60).to_i
+        when :score
+          value.respond_to?(:*) ? "#{(value*100).round(0)}%" : ''
+        else
+          value
+        end
+      end
+
+      def self.row_contains_requested_columns?(row, requested_columns)
+        (row.keys & requested_columns).length == requested_columns.length
+      end
```

Comment:
If, somehow, `requested_columns` ended up having duplicate key names, this comparison will fail as `&` will collapse duplicates.  My bet is that we have no reason to believe that this will ever happen, so no need to make a change here, but I wanted to flag it to make sure we'd at least considered the possibility.